lib/vendors/axis-communications/sv_vendor_axis_communications.c

```diff
@@ -872,7 +890,17 @@ sv_vendor_axis_communications_set_attestation_report(signed_video_t *sv,
       sv->onvif = onvif_media_signing_create(sv_codec_to_ms_codec(sv->codec));
     }
   }
-
+  if (sv->sign_data) {
+    if (sv->onvif) {
+      // Set the signing key pair if sign_data is not available
```

Comment:
I think you should explain why it is possible to set the signing key pair if `sign_data` is available.
Also, there is a typo in the comment. I think the "not" should be removed.